src/components/application_manager/src/app_service_manager.cc

```diff
@@ -142,10 +142,12 @@ smart_objects::SmartObject AppServiceManager::PublishAppService(
     for (auto it = embedded_services.begin(); it != embedded_services.end();
          ++it) {
```

Comment:
maybe range-based

Reply:
@SNiukalov fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/f4e32226bda388a572a9edc282e74636a66ad925